packages/babel-preset/src/assets/normalizeStyleRules.ts

```diff
@@ -23,15 +23,24 @@ export function normalizeStyleRule(
   }
 
   const result = parseStringWithUrl(ruleValue);
-  // Quotes in URL are optional, so we can also normalize them
+  // Quotes in URL are optional, we can also normalize them,
+  // so two rules are treated as same when they only differ in quotations.
   // https://www.w3.org/TR/CSS2/syndata.html#value-def-uri
-  const url = result.url.replace(/['|"](.+)['|"]/, '$1');
+  const url = result.url.replace(/^(['|"])(.+)\1$/, '$2');
 
-  if (url.startsWith('data:')) {
+  if (url.startsWith('data:') || url.startsWith('http:') || url.startsWith('https:')) {
     return `${result.prefix}${url}${result.suffix}`;
   }
 
-  return `${result.prefix}${normalizeAssetPath(path, projectRoot, filename, url)}${result.suffix}`;
+  // Handle `filter: url(#id)` and `filter: url(./a.svg#id)`
+  const [pathname, hash] = url.split('#');
+  if (!pathname) {
+    return `${result.prefix}${url}${result.suffix}`;
+  }
+
+  return `${result.prefix}${normalizeAssetPath(path, projectRoot, filename, pathname)}${hash ? `#${hash}` : ''}${
+    result.suffix
+  }`;
```

Comment:
So, only for assets that are in this branch we are need post-processing, right? 🤔 

Can we add a special prefix `griffel||` or special symbol `\f` to identify them in a simpler way in `replaceAssetsWithImports`?

Reply:
I think the current approach (using `isAssertUrl` at both places) is clear enough.